kahuna/public/js/components/gr-photoshoot/gr-photoshoot.html

```diff
@@ -12,7 +12,7 @@
           e:ng-class="{'image-info__editor--error': $error,
                        'image-info__editor--saving': photoshootEditForm.$waiting,
                        'text-input': true}"
-          e:placeholder="describing a set of images"
+          e:placeholder="describing a set of images for syndication purposes"
```

Comment:
Echoing @paperboyo's email, photoshoots can be used for more than syndication purposes, maybe an `e.g.` would work? `describing a set of images, e.g. for syndication purposes`